SpatialGDK/Source/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -56,11 +56,13 @@ struct FLocatorConfig : public FConnectionConfig
 		const TCHAR* CommandLine = FCommandLine::Get();
 
 		FParse::Value(CommandLine, TEXT("projectName"), ProjectName);
+		FParse::Value(CommandLine, TEXT("deploymentName"), DeploymentName);
```

Comment:
This is for the `FLocatorConfig` so I believe it's relevant no?

Reply:
If it's only relevant for `FLocatorConfig` then it should be in that class.

But this isn't needed because we dynamically grab the deployment list. Look at `USpatialWorkerConnection::ConnectToLocator()`

Reply:
Yup, this line should be in the constructor for `FLocatorConfig` (I think the original line is lost b/c of a force push...)

Also `ConnectToLocator()` just connects to the first deployment that you get from the list. The point of this flag was to search that list for a deployment with the corresponding name and connect to it specifically, otherwise to default to the first deployment in the project.